src/client/pages/VergunningDetail/BZB.tsx

```diff
@@ -4,10 +4,59 @@ import InfoDetail, {
 } from '../../components/InfoDetail/InfoDetail';
 
 import type { BZB as BZBVergunning } from '../../../server/services/vergunningen/vergunningen';
+import { useAppStateGetter } from '../../hooks';
+import { MyNotification } from '../../../universal/types';
+import { InnerHtml, LinkdInline } from '../../components';
+
+function ExpirationdNotifications({ id }: { id: string }) {
```

Comment:
Staat hier een d teveel in de component naam? _ExpirationNotifications_ zou ik verwachten.